core/model/impl/src/main/kotlin/AssistedInjectFactoryModelImpl.kt

```diff
@@ -87,6 +87,8 @@ internal class AssistedInjectFactoryModelImpl private constructor(
     }
 
     override fun validate(validator: Validator) {
+        super.validate(validator)
```

Comment:
Maybe it would be better to add aditional checks in [global validation traverse](https://github.com/yandex/yatagan/blob/main/validation/impl/src/main/kotlin/validation.kt#L59) instead of changing each validate method (and possibly forgetting to add super call)

Reply:
Done. Yes, it looks way better this way, thanks for the suggestion!